priam/src/main/java/com/netflix/priam/utils/TokenManager.java

```diff
@@ -45,9 +72,10 @@
          * unit test failures.
          */
         Preconditions.checkArgument(position >= 0, "position must be >= 0");
-        return MAXIMUM_TOKEN.divide(BigInteger.valueOf(size))
+        return tokenRangeSize.divide(BigInteger.valueOf(size))
```

Comment:
The original algorithm for "random token space" uses the max token value to derive the initial token.  With this commit, the algorithm changed to use the range.  Is this not changing the original algorithm?